tests/api/test_eval.py

```diff
@@ -5,12 +5,16 @@ class TestEvalResource(SnekAPITestCase):
     PATH = "/eval"
 
     def test_post_valid_200(self):
-        body = {"input": "foo"}
-        result = self.simulate_post(self.PATH, json=body)
-
-        self.assertEqual(result.status_code, 200)
-        self.assertEqual("output", result.json["stdout"])
-        self.assertEqual(0, result.json["returncode"])
+        cases = [
+            {"args": ["-c", "print('output')"]},
+            {"input": "print('hello')"},
```

Comment:
We should also test a case where both input and args are provided.

Reply:
Added in [95e3e18](https://github.com/python-discord/snekbox/pull/159/commits/95e3e18ac93709257840f8e5c36e6543678931f5)